scalafmt-tests/src/test/resources/unit/Case.case

```diff
@@ -81,3 +81,15 @@ case q"""...${stats: Seq[Stat]} """ =>
 case  List(xs @ _*) => a
 >>>
 case List(xs @ _*) => a
+<<< #2258 Evenly align case values
+newlines.source = keep
+includeCurlyBraceInSelectChains = true
+continuationIndent.caseSite = 3
+===
+case _: Aaaaaa |
+    _: Bbbbbb |
+    _: Cccccc =>
```

Comment:
you need to add plenty of tests, with various level of complexity (including `if` guards, case body in a block or without, etc.).

Reply:
Added, please check.

Reply:
i don't think you added any tests where bodyBlock is true

Reply:
Can you help me on this?

I tried…
```
case {
  _: SomeClass
} => ???
```
…but it fails with syntactic error.

Reply:
i think `bodyBlock` refers to `{}` after the arrow

Reply:
Added